portal/src/presentation/style.scss

```diff
@@ -0,0 +1,20 @@
+@import "~@fremtind/jkl-core/variables/all";
+
+.jkl-s {
```

Comment:
This is an experimental API we should improve and consider adopting in core, to make adding margins and gutters for component easier